src/data/markdown/docs/01 guides/05 Testing Guides/04 Large-scale k6 testing.md

```diff
@@ -0,0 +1,484 @@
+---
+title: "Hardware-sizing for large-scale k6 tests"
```

Comment:
Maybe "Hardware requirements for large-scale k6 tests" or "Determining hardware requirements ..."?

Or "resource requirements" considering the article also discusses OS tweaks?

Reply:
I vote for hardware requirements

Reply:
Change per your recommendation :+1: